_data/sites.yml

```
@@ -133,6 +133,12 @@
   method: mediaquery
   last_checked: 2023-04-12
 
+  - domain: cri.cl
+    url: https://cri.cl/
+    method: darkonly
+    last_checked: 2024-03-28
+
```

```suggestion
- domain: cri.cl
  url: https://cri.cl/
  method: darkonly
  last_checked: 2024-03-28

```
Whoops, messed it up, I'm on my phone. 😅